script/default-site

```diff
@@ -2,16 +2,16 @@
 # Runs the `jekyll new` command and builds the default site as a sanity check
 
 set -e
-
+echo "PLATFORM:" $(uname -s)
 echo "$0: setting up tmp directory"
 mkdir -p ./tmp
 rm -Rf ./tmp/default-site
 
-workdir=$(pwd)
-
 echo "$0: creating new default site"
-bundle exec exe/jekyll new tmp/default-site
+bundle exec jekyll new tmp/default-site
+
```

Comment:
It seems like the executable exe/jekyll is unnecessary. Appveyor & Travis both, are able to run Jekyll as available at source . 
Do correct me, if it really is necessary